pkgs/applications/science/biology/migrate/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchurl, zlib, gcc, glibc, openmpi }:
+
+stdenv.mkDerivation rec {
+  version = "3.7.2";
+  pname = "migrate";
+
+  src = fetchurl {
+    url = "https://peterbeerli.com/migrate-html5/download_version3/${pname}-${version}.src.tar.gz";
+    sha256 = "0mb8b0jrjb83bp2f6if2gdwz5f0nbysc1vchn5bfa4jhxfgscr53";
+  };
+
+  buildInputs = [ zlib glibc openmpi ];
+  nativeBuildInputs = [ gcc ];
```

Comment:
This is unnecessary on linux at least. If this program cannot be compiled with clang, use gccStdenv instead of stdenv.